lib/utils/segment.js

```diff
@@ -37,7 +25,7 @@ function track(payload) {
   if (TRACKING_IS_DISABLED) {
     return BbPromise.resolve();
   }
-  return request('https://api.segment.io/v1/track', payload);
+  return request('https://tracking.serverlessteam.com/v1/track', payload);
```

Comment:
Thanks for the update. That make sense @alexdebrie 👍 Not sure if we need a Sentry one just yet (/cc @eahefnawy).